src/frameName.cpp

```diff
@@ -168,6 +169,22 @@ void FrameName::javaMethodName(jmethodID method) {
     char* class_name = NULL;
     char* method_name = NULL;
     char* method_sig = NULL;
+    jint modifiers = 0;
+
+    // Based on: https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-4.html#:~:text=Table%C2%A04.5.%C2%A0Method%20access%20and%20property%20flags
+    std::map<int, std::string> access_flags;
+    access_flags[0x0001] = "public";
+    access_flags[0x0002] = "private";
+    access_flags[0x0004] = "protected";
+    access_flags[0x0008] = "static";
+    access_flags[0x0010] = "final";
+    access_flags[0x0020] = "synchronized";
+    access_flags[0x0040] = "bridge";
+    access_flags[0x0080] = "varargs";
+    access_flags[0x0100] = "native";
+    access_flags[0x0400] = "abstract";
+    access_flags[0x0800] = "strict";
+    access_flags[0x1000] = "synthetic";
```

Comment:
These are all consts - can be made a const of the class instead of code in the method?